frontend/src/metabase-lib/v1/expressions/config.ts

```diff
@@ -326,23 +326,41 @@ const CONVERSION = defineClauses(
       description: () => t`Converts a datetime string or bytes to a datetime.`,
       hasOptions: true,
       validator(_value, mode) {
-        const validModes = ["iso", "simple", "isobytes", "simplebytes"];
+        // missing mode is fine
+        if (mode === undefined) {
+          return;
+        }
+
+        const validModes = [
+          "iso",
+          "simple",
+          "isobytes",
+          "simplebytes",
+          "unixseconds",
+          "unixmilliseconds",
+          "unixmicroseconds",
+          "unixnanoseconds",
+        ];
+        const normalizedMode = mode.toLowerCase();
 
-        if (mode !== undefined && !validModes.includes(mode)) {
-          return t`Invalid mode: ${mode}`;
+        // mode is valid after normalization
+        if (validModes.includes(normalizedMode)) {
+          return;
         }
+
+        return t`Invalid mode: ${mode}`;
       },
       args: () => [
         {
           name: t`value`,
           type: "expression",
-          description: t`The string or bytes to convert to a datetime.`,
+          description: t`The string, bytes, or number to convert to a datetime.`,
           example: "2025-03-20 12:45:04",
         },
         {
           name: t`mode`,
           type: "string",
-          description: t`The mode indicating the format. One of: "simple", "iso", "simplebytes", "simpleiso". Default is "iso".`,
+          description: t`The mode indicating the format. One of: "simple", "iso", "simpleBytes", "isoBytes", "unixSeconds", "unixMilliseconds", "unixMicroseconds", "unixNanoseconds". Default is "iso".`,
```

Comment:
Please mark code-like stuff as such using <code>\\`</code>
```suggestion
          description: t`The mode indicating the format. One of: \`"simple"\`, \`"iso"\`, \`"simpleBytes"\`, \`"isoBytes"\`, \`"unixSeconds"\`, \`"unixMilliseconds"\`, \`"unixMicroseconds"\`, \`"unixNanoseconds"\`. Default is \`"iso"\`.`,
```